docs/resourse-cleanup.md

```diff
@@ -0,0 +1,64 @@
+# Stale Resource Cleanup
+
+If the PVC is created with storage class which is having the `reclaimPolicy``
+as `Retain` will not delete the PV object , backend omap metadata and backend image.
+Manual deletion of PV will result in stale omap keys, values , cephfs subvolume and rbd image.
+You need to cleanup metadata and image separately.
+
+## Steps
+
+
+### 1. Get volume ID from PV
+
+* '[$]kubectl get pvc pvc_name -n namespace -owide' to get pv_name <br>
```

Comment:
Instead of the ' quote, use a single backtick `.
No need to add `<br>`, use markdown style completely. The following lines should be indented by (2 or 4?) spaces (including the triple backticks). You can use https://hackmd.io for testing the formatting.

Reply:
done